.github/ISSUE_TEMPLATE/icon_rebrand.yml

```diff
@@ -1,31 +1,24 @@
-name: Request icon design change
-description: Request changing the design of an existing icon in Lawnicons
-title: "[Icon Rebrand] "
-labels: ["icon rebrand"]
+name: Change an outdated icon
+description: Request updating an icon if it is outdated or applied incorrectly.
+labels: ["icon change"]
 body:
   - type: textarea
     id: outdated-icon-list
     attributes:
-      label: List of outdated icons
+      label: Outdated icons
       description: |
-        Please follow the following format regarding the outdated icons:
-
-        ```
-        App Name
-        (image to new icon)
+        Please specify what will change:
+        
+        **Old App Name**
+        The [link to the outdated icon](https://github.com/LawnchairLauncher/lawnicons/tree/develop/svgs).
+        ↓
+        **New App Name**
```

Comment:
This bit personally seems confusing. I know sometimes apps change names while rebranding, but I think it would be better to have a separate template for that.

Reply:
>  I know sometimes apps change names while rebranding, but I think it would be better to have a separate template for that.

Hmm, I'll try to change it. It is rather important to find out the old name here, because many icons have several apps linked to them.

Reply:
If you were referring to a separate template for the new name, then this is unnecessary. It happens too rarely and is easy to fix if there is a mistake.